packages/spec-parser/src/specParser.ts

```diff
@@ -376,6 +376,28 @@ export class SpecParser {
           existingPluginManifestInfo
         );
 
+      const functions = apiPlugin.functions;
+
+      if (functions) {
+        const adaptiveCardFolder = path.join(path.dirname(pluginFilePath), "adaptiveCards");
```

Comment:
the adaptiveCards folder is hard coded?

Reply:
Yes, for our template, it is by default inside appPackage/adaptiveCards, user can change it by update the reference inside ai-plugin.json file